draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -801,12 +803,12 @@ A Primary ID can be constructed for each message which is composed of the follow
 1. Transport
 1. DNS Message ID
 
-### Secondary ID (optional)
+### Secondary ID
 
 If present, the first question in the Question section is used as a secondary ID
-for each message. Note that there may be well formed DNS queries that have a
-QDCOUNT of 0, and some responses may have a QDCOUNT of 0
-(for example, RCODE=FORMERR or NOTIMP)
+for each message. Note that some queries and some responses may have a QDCOUNT of 0
```

Comment:
There were a couple of steps between the two, so it wasn't a straightforward replacement. I've restored the original wording.